### PR TITLE
build: include mycetozoa

### LINKED ISSUE #9: Make Python Script Build Phylum Mycetozoa
Have to figure out how to add only the phylum in addition to the rest of the data model. I think I can build the preset json out more, with the top taxon being life, then having the normal fungi, then having a separate branch going straight from Protist to Mycetozoa

### PR DESCRIPTION
**Pull request by mrozanoff**
### Description: ✍️
* Updated Python code to update JSON structure. Updated JS to reflect the increase in taxon levels in the JSON file.
* Seems to be a possible bug with the sorting of locales (Ecuador has a count of 2 and is at the top), but that may be fixed automatically since some of the keys are not matching the correct taxons. Not a priority.

### Issue(s): 🎟️
* #9

### CODE REVIEW

File: typahead_genera.html [hunk 1]
```diff
@@ -196,7 +196,7 @@ <h1>Search For Fungus Key</h1>
         async function loadJSON() {
             try {
                 // Fetch the JSON file (replace 'data.json' with your file path)
-                const response = await fetch('./fungi_taxonomy.json');
+                const response = await fetch('./fungi_taxonomyTESTING.json');
```
Comment (Elaniobro) [inline typahead_genera.html]: should this be `fungi_taxonomy.json`?

File: typahead_genera.html [hunk 2]
```diff
@@ -252,12 +251,11 @@ <h1>Search For Fungus Key</h1>
             }
         }
 
-
-
         // Recursive search function
         function searchRecursive(obj, searchTerm, results) {
-            if (obj == null) return; // Check if obj is null
 
+            if (obj == null) return; // Check if obj is null
```
Comment (Elaniobro) [inline typahead_genera.html]: Lets make this read: `if (obj === null) return;` 

`===` is safer in .js

File: typahead_genera.html (hunk 1 above)
Comment (mrozanoff) [reply typahead_genera.html]: yes, I must have forgotten to switch it, thanks

**Review by Elaniobro — APPROVED** [review]
:shipit: - Looks good.

**Pull request merged**